public/_locales/en/messages.json

```diff
@@ -646,6 +646,9 @@
     "bracketNow": {
         "message": "(Now)"
     },
+    "bracketStart": {
+        "message": "(Start)"
+    },
```

Comment:
I know we have other bracket ones, but I think it would be best to add the brackes in the code instead of in the localisation

Reply:
are there localizations for "start"?

Reply:
made key for "start", added brackets in code